compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -398,7 +398,8 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
                     &self,
                     exclusive_range_pattern,
                     pattern.span,
-                    "exclusive range pattern syntax is experimental"
+                    "exclusive range pattern syntax is experimental",
+                    "use an inclusive range pattern, like N..=M"
```

Comment:
This could also be made an automatically applicable suggestion using the right `N` and `M` values, but this is already an improvement, so let's land it.